### PR TITLE
Add extra label to PVC

### PR DESCRIPTION
**Pull request by emitamago**
Adding new extra company/account label to pvc

### CODE REVIEW

File: pkg/operator/vttablet/pvc.go [hunk 1]
```diff
@@ -24,7 +24,13 @@ import (
 )
 
 // NewPVC creates a new vttablet PVC from a Spec.
+// TODO: "Handle the case when labels are removed from ExtraLabels"
```
Comment (enisoc) [inline pkg/operator/vttablet/pvc.go:27]: We don't usually put quotes around the TODO comment. Also, let's move this to right before the point where the fix would need to go, which is line 49:

```go
	// update extra labels
	// TODO: Handle the case when labels are removed from ExtraLabels.
	update.Labels(&obj.Labels, spec.ExtraLabels)
```

File: pkg/operator/vttablet/pvc.go [hunk 2]
```diff
@@ -24,7 +24,13 @@ import (
...
 func NewPVC(key client.ObjectKey, spec *Spec) *corev1.PersistentVolumeClaim {
+	// Store labels in labels obj because we need to add extra label and avoid mutating spec.Labels value
+	labels := map[string]string{}
```
Comment (enisoc) [inline pkg/operator/vttablet/pvc.go:30]: We should change `Labels: spec.Labels` below to use this var instead.

**Review by enisoc — COMMENTED** [review]
Would you mind applying the same fix to `operator/etcd/pvc.go` as well? It should be the same change.

File: pkg/operator/vttablet/pvc.go (hunk 1 above)
Comment (emitamago) [reply pkg/operator/vttablet/pvc.go]: moved to recommended place and removed quote

File: pkg/operator/vttablet/pvc.go (hunk 2 above)
Comment (emitamago) [reply pkg/operator/vttablet/pvc.go:29]: sorry I do not why it was like this. fixed to use this var

**Review by emitamago — COMMENTED** [review]
thank you so much for your review. 

question is that why you recommend to apply same fix to etcd/pvc.go as well?

**Comment by enisoc** [comment]
> question is that why you recommend to apply same fix to etcd/pvc.go as well?

The operator manages etcd PVCs the same way it does vttablet PVCs, so we should be consistent in applying ExtraLabels in both cases.

**Review by enisoc — APPROVED** [review]
LGTM

**Pull request merged**